### ISSUE TITLE
Regarding vue-fontawsome and sizing

### ISSUE THREAD

**Issue by hjelperne** [yaminncco/vue-sidebar-menu#60]
Hi,
Thanks for fixing this. Great!
I have now updated to use the vue-fontawesome icons instead, but I am experiencing the icons being much larger now - than previously (see attached image).
![image](https://user-images.githubusercontent.com/342842/60709492-d7c7cf00-9f10-11e9-98b7-fde6a7189ac8.png)

Code ref:
`                    {
                        href: '/',
                        title: 'CRM',
                        icon:  {
                            element: 'font-awesome-icon',
                            attributes: {
                                icon: 'medal',
                            }
                        },
                        disabled: true
                    },

                    {
                        href: '/',
                        title: 'CRM',
                        icon: 'fas fa-medal',
                        disabled: true
                    },`

Is the any way to control the sizing? 
Ref: https://github.com/FortAwesome/vue-fontawesome#user-content-basic

It would be really nice to have the icons smaller (like before ~ when using font-awsome directly).

Great work!

_Originally posted by @hjelperne in https://github.com/yaminncco/vue-sidebar-menu/issues/52#issuecomment-508682067_

**Comment by hjelperne** [comment]
By the way - currently I use the following scss override to acheive smaller icons (like before):

`.v-sidebar-menu .vsm-item.first-item>.vsm-link>.vsm-icon {
        height: 16px !important; /* override height */
        margin-top: 6px; /* acheive correct vertical alignment */
    }`

**Comment by hjelperne** [comment]
And finally I noticed one more issue with switching to vue-fontawsome icons. 
The 2 expand icons for respectively: 
1) "submenus expand/collapse" icon (vsm-arrow) 
2) collpase menu icon (content: '\F337'; on button) 
become "squares" as they are no longer found when I remove the: 
**import '@fortawesome/fontawesome-free/css/all.css'**
which is basically the entire point of using vue-fontawsome instead.. (more compact).

Hope you can help.

Thanks.

**Comment by yaminncco** [comment]
hi @hjelperne slots are available in v3.11 you can use them to customize icons

```
<sidebar-menu>
    <font-awesome-icon slot="collapse-icon" :icon="'arrows-alt-h'"></font-awesome-icon>
    <font-awesome-icon slot="dropdown-icon" :icon="'angle-right'"></font-awesome-icon>
</sidebar-menu>
```
and about size you can add some padding to `.vsm-arrow` to fix it

**Comment by hjelperne** [comment]
Thank you!
